tests/src/python/test_qgsrectangle.py

```diff
@@ -252,6 +252,15 @@ def testToBox3d(self):
         self.assertEqual(box.yMaximum(), 0.3)
         self.assertEqual(box.zMaximum(), 0.5)
 
+    def testOperators(self):
+        rect1 = QgsRectangle(10, 20, 40, 40)
+        rect2 = rect1 + QgsVector(3, 5.5)
+        assert rect2 == QgsRectangle(13, 25.5, 43, 45.5), "QgsRectangle + operator does no work"
+
+        # Subtracting the center point, so it becomes zero.
+        rect1 -= rect1.center() - QgsPointXY(0, 0)
```

Comment:
thanks for explanation.
it would be nice to do `QgsRectangle(1, 2, 3, 4) - [1,2]` in python
Anyway great job as it is!